### DISCUSSION TITLE
Actions not being run, waiting over a day for a job to be picked up

### DISCUSSION THREAD

**Discussion by davidgs** [community/community#56941] [Actions] [answered]
### Select Topic Area

Bug

### Body

I have a GitHub Action that has been waiting for 1 day, 11hours for a runner to pick up the job. It's a fairly simple action to simply do a hugo build, but after a day and a half, it's still sitting there with nothing. 

I already stopped it after the first 8 hours and restarted it, hoping that would work, but nope! So in reality, I'm 1 day, 16 hours into this with no end in sight.

**Comment by airtower-luna** [comment, marked as answer]
Does the job happen to use a `ubuntu-18.04` runner? I've seen a few people ask about stuck jobs recently and it turned out they were using a runner that [doesn't exist any more](https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/). :sweat_smile:

**Comment by davidgs** [comment]
That's it. Changed to `ubuntu-latest` and it kicked loose!
